TechnicalDocuments/FastlaneMatch.md

```diff
@@ -0,0 +1,147 @@
+Fastlane Match
+==============
+
+Fastlane Match is a tool for managing certificates and provisioning profiles centrally. It downloads certificates and profiles from a GitHub repository or Google Cloud Storage bucket. For safety, centrally stored files are encrypted.
+
+## Installing Certificates and Provisioning Profiles Locally
+
+For day to day development work it usually suffices to navigate to the project directory in a terminal shell and run the command
+
+```shell
+ $ fastlane match development --read-only
+```
+
+Certificates and provisioning profiles for enterprise builds (Hockey App) or release builds (Testflight and the App Store) can also be downloaded. Normally, this is not needed but might be necessary if you want to run UI tests locally or work on updating our fastlane build scripts.
+
+To install certificates and provisioning profiles for creating a Testflight or App Store build locally, run the command
+
+```shell
+ $ fastlane match release git_branch:"release team id" --read-only
```

Comment:
```suggestion
 $ fastlane match release git_branch:"release team id" --readonly
```

Reply:
It must be `enterprise` for HockeyApp builds (which are using our enterprise account) or `appstore` for TestFlight/AppStore builds

```suggestion
 $ fastlane match appstore git_branch:"appstore team id" --read-only
```